cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard-modal.html

```diff
@@ -24,6 +24,12 @@ <h5 ng-if="action.name" class="modal-title pull-left">
       <small>{{action.description}}</small>
     </p>
   </h5>
+  <span class="text-success"
+        ng-if="errorCount === 0">No Errors found.</span>
+  <span class="text-danger"
+        ng-if="errorCount > 0">
+    {{errorCount}}
+    error{{errorCount === 1 ? '': 's'}} found.</span>
```

Comment:
Can we use ng-pluralize here?

Reply:
TIL there's ng-pluralize :smile: thanks!